lib/daru/dataframe.rb

```diff
@@ -263,6 +264,74 @@ def [](*names)
       dispatch_to_axis axis, :access, *names
     end
 
+    # Retrive rows by positions
+    # @param [Array<Integer>] *positions positions of rows to retrive
+    # @return rows specified by positions
+    def at *positions
+      if positions.size == 1
+        return Daru::Vector.new @data.map { |vec| vec.at(*positions) },
+          index: @vectors
+      else
+        new_rows = @data.map { |vec| vec.at(*positions) }
+        return Daru::DataFrame.new new_rows,
+          index: @index.at(*positions),
+          order: @vectors
+      end
+    end
+
+    # Set rows by positions
+    # @param [Array<Integer>] positions positions of rows to set
+    # @vector [Array, Daru::Vector] vector vector to be assigned
+    def row_at_set positions, vector
+      vector =
+        if vector.is_a? Daru::Vector
+          vector.reindex @vectors
+        else
+          Daru::Vector.new vector
+        end
+
+      raise SizeError, 'Vector length should match row length' if
+        vector.size != @vectors.size
+
```

Comment:
> 'positions' is repeated in the comments above.

First instance of positions is the name of the argument and second instance of positions is for its description.

> BTW if we already have a method for this in the form of row.set_at why do you even need this method?
You can modify DataFrame#at in the same way that DataFrame#[] works: pass :row as a second argument.

I didn't get it. I would still need `set_row_at` even if I pass `at` with `:row`. For example there is `access_row` and `access_vector`.